src/test/java/org/jenkinsci/plugins/workflow/flow/DurabilityBasicsTest.java

```diff
@@ -84,7 +85,7 @@ public void managePermissionShouldAccessGlobalConfig() throws Throwable {
 
             try (ACLContext c = ACL.as(User.getById(USER, true))) {
                 Collection<Descriptor> descriptors = Functions.getSortedDescriptorsForGlobalConfigUnclassified();
-                assertTrue("Global configuration should not be accessible to READ users", descriptors.size() == 0);
+                assertEquals("Global configuration should not be accessible to READ users", 0, descriptors.size());
```

Comment:
or probably something like (plus imports)
```suggestion
                assertThat("Global configuration should not be accessible to READ users", descriptors, empty());
```

Reply:
Good catch!